client/client/app/components/ngbBlastSearchPanel/ngbBlastWholeGenomeView/ngbBlastWholeGenomeView.scss

```diff
@@ -0,0 +1,16 @@
+.whole-genome-md-dialog {
+  min-height: 400px;
+  height: 80vh;
+  background-color: #ffffff;
+  overflow-y: hidden;
+}
+
+#whole-genome-view-container {
```

Comment:
Let's use classes instead of identifiers

Reply:
Done!